Avoid error message

  Error in file "/usr/local/share/applications/mars.desktop":
  "application" is an invalid MIME type ("application" does not contain
  a subtype)

--- resources/mars.desktop.orig	Tue Aug 16 12:56:27 2016
+++ resources/mars.desktop	Tue Aug 16 12:56:41 2016
@@ -8,4 +8,3 @@ Icon=mars
 Terminal=false
 Categories=Game;ActionGame; 
 StartupNotify=false
-MimeType=application;
